tests/kernel/msgq/msgq_usage/testcase.yaml

```diff
@@ -0,0 +1,4 @@
+tests:
+  kernel.message_queue_usage:
+    tags: kernel userspace
+    platform_exclude: native_posix native_posix_64 nrf52_bsim
```

Comment:
why are those being excluded?

Reply:
Those new created threads have while(1) or while(condition) which "condition" is update in other threads,
on native_posix platform, that cause creator thread can't be wake up, as the reason listed in https://docs.zephyrproject.org/latest/boards/posix/native_posix/doc/index.html#important-limitations, I exclude native_posix and native_posix_64. Excluding nrf52_bsim because CI test error.
Now I remove platform_exclude clause and add k_msleep() in test code.

Reply:
@nashif refined, now all case passed without exclude those platforms.